src/manager.ts

```diff
@@ -18,6 +18,10 @@ export class Manager {
         this.extension = extension
     }
 
+    get rootDir() {
+        return path.dirname(this.rootFile)
+    }
+
```

Comment:
added this property to save having to do this common operation elsewhere in the parser